lib/checks/forms/fieldset.js

```diff
@@ -50,28 +50,62 @@ function checkARIAGroup(group, name) {
 	return true;
 }
 
+function checkTable (group, nodes) {
```

Comment:
I thought about that, but decided against it. Our `isDataTable` method is fairly conservative, only flagging things as such if it's absolutely certain. If we fail this test every time that method isn't certain about it, we risk false positives. That's why instead of doing that, I just checked if `role=presentation` is on it.

As for headers, I think it's fairly typical for a column of checkboxes to not have a header. For example, the issues table in Comply only has an empty th above the column of checkboxes.